lua/tests/gmod/classes/file/WriteFloat.lua

```diff
@@ -0,0 +1,26 @@
+return {
+    groupName = "File:WriteFloat",
+
+    cases = {
+        {
+            name = "Exists on the File meta table",
+            func = function()
+                local meta = FindMetaTable( "File" )
+                expect( meta.WriteFloat ).to.beA( "function" )
+            end
+        },
+
+        {
+            name = "Returns the right value",
+            func = function()
+                local a = GetTestFile( "WriteFloat", true )
+                a:WriteFloat( 1.0000001 )
+                a:Close()
+
+                local b = GetTestFile( "WriteFloat" )
+
+                expect( b:ReadFloat() ).to.equal( 1.0000001192092895507812 )
+            end
+        },
```

Comment:
Is this expected value always going to be imprecise in the same way? i.e. across base branch and x86-64?

```suggestion
        {
            name = "Writes a Float correctly",
            func = function()
                local a = GetTestFile( "WriteFloat", true )
                a:WriteFloat( 1.0000001 )
                a:Close()

                local b = GetTestFile( "WriteFloat" )

                expect( b:ReadFloat() ).to.equal( 1.0000001192092895507812 )
            end
        },
```